app/populate/data/graduation.js

```diff
@@ -0,0 +1,49 @@
+module.exports = function () {
+  return  [{      
+      "curso": "Bacharelado em Ciência e Tecnologia",
+      "grade": "2017",
+      "credits_total": 190,
+      "free_credits_number": 43,
+      "limited_credits_number": 57,
+      "locked": true,
+      "mandatory_credits_number": 90,
+      "creditsBreakdown": [{
+          "year": 1,
+          "quad": 1,
+          "choosableCredits": 0
+      }, {
+          "year": 1,
+          "quad": 2,
+          "choosableCredits": 0
+      }, {
+          "year": 1,
+          "quad": 3,
+          "choosableCredits": 0
+      }, {
+          "year": 2,
+          "quad": 1,
+          "choosableCredits": 0
+      }, {
+          "year": 2,
+          "quad": 2,
+          "choosableCredits": 12
+      }, {
+          "year": 2,
+          "quad": 3,
+          "choosableCredits": 16
+      }, {
+          "year": 3,
+          "quad": 1,
+          "choosableCredits": 24
+      }, {
+          "year": 3,
+          "quad": 2,
+          "choosableCredits": 24
+      }, {
+          "year": 3,
+          "quad": 3,
+          "choosableCredits": 24
+      }]
+    }
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555380)
```suggestion
  }
```